WPMedia/fusion-news-theme-blocks
# Tmedia 397 new account creation

## Description
Create account 

## Jira Ticket
- [TMEDIA-397](https://arcpublishing.atlassian.net/secure/RapidBoard.jspa?rapidView=875&modal=detail&selectedIssue=TMEDIA-397&assignee=5e387d6a1b1d910e5dfd7a9b)

## Acceptance Criteria
User can create an account

User is logged in after account creation, returning to article or defined URL

Form validation / error states -> deferred for @nschubach's error state

T&C text is out of scope for this ticket – will be handled separately pending more investigation by product

## Test Steps

*confirm pass is deferred*

1. Checkout this branch `git checkout TMEDIA-397-new-account-creation`
2. Setup arc header field and apiOrigin per https://paper.dropbox.com/doc/Themes-Subscriptions-Proof-of-Concept--BQKuqQ4hMJt81bP3Y9xr9GhZAg-aHjkUr5TYxztGDAPBf4eb
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block`
4. Go to page http://localhost/pf/account/sign-up/?_website=the-sun
5. Type in username of at least 5 chars - 100 chars 
6. Type password of at least 6 chars 
7. Put in valid email with @ that hasn't been signed in before. You can modify with user+11@email.com if necessary
8. Get redirected To article 

## Effect Of Changes
### Before
- could not sign in 

### After
- Can sign in

## Dependencies or Side Effects

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
